src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -100,6 +114,16 @@ const LaunchAnalysisModal = ({
     ]),
     div({ style: { color: colors.danger(), overflowWrap: 'break-word' } }, [
       h(Fragment, wrappableOnPeriods(launchError))
+    ]),
+    div(['Output files will be saved as workspace data in:']),
+    div({ style: { margin: '1rem' } }, [
+      location ? h(Fragment, [span({ style: { marginRight: '0.5rem' } }, [flag]),
+        span({ style: { marginRight: '0.5rem' } }, [regionDescription]),
+        h(InfoBox, [
+          p(['Cromwell will instruct its job runners to delocalize call outputs to your workspace bucket in this region.']),
```

Comment:
nit: Is 'delocalize' a commonly understood word, or could we say something more straightforward like 'write'?